src/tools/mmaps_generator/MapBuilder.cpp

```diff
@@ -1015,7 +1016,7 @@ namespace MMAP
         config.maxVertsPerPoly = DT_VERTS_PER_POLYGON;
         config.cs = tileConfig.BASE_UNIT_DIM;
         config.ch = tileConfig.BASE_UNIT_DIM;
-        config.walkableSlopeAngle = 80;
+        config.walkableSlopeAngle = 85;
```

Comment:
just wondering why changed this.

Reply:
70° and all filters
![image](https://user-images.githubusercontent.com/1153754/84574141-ea938680-ada4-11ea-9f62-b8810f57b345.png)

80° and all filters
![image](https://user-images.githubusercontent.com/1153754/84574159-f5e6b200-ada4-11ea-8ae6-faa4002032fb.png)

80° and exclude ledge span filter
![image](https://user-images.githubusercontent.com/1153754/84574170-fe3eed00-ada4-11ea-9ad5-2c3709df4582.png)

85° and all filters
![image](https://user-images.githubusercontent.com/1153754/84574179-07c85500-ada5-11ea-8c1c-5b0cf91761a2.png)

85° and exclude ledge span filter
![image](https://user-images.githubusercontent.com/1153754/84574190-1151bd00-ada5-11ea-9bb8-2f011383a35b.png)